packages/chat-core-aws-connect/THIRD-PARTY-NOTICES

```diff
@@ -1,48 +1,2876 @@
+The following NPM packages may be included in this product:
```

Comment:
this is a lot of dependency changes even though you didn't change anything in package.json, can you verify they are actually relevant for chat-core-aws-connect?